cellpack/autopack/Environment.py

```diff
@@ -1949,7 +1961,7 @@ def getTotalNbObject(self, allIngredients, update_partner=False):
         totalNbIngr = 0
         for ingr in allIngredients:
             if ingr.type == "Grow":
-                totalNbIngr += int(ingr.left_to_place * (ingr.length / ingr.uLength))
+                totalNbIngr += int(ingr.left_to_place * (ingr.length / ingr.length))
```

Comment:
I think this should still be `uLength`?

Reply:
yeah these are changes from my recipe loader branch, you're right. I changed this to `unit_length` though. @rugeli I'll make this change